scripts/README.md

```diff
@@ -0,0 +1,36 @@
+## Syncing downstream OLM
+
+All of the staged repositories live in the top level `staging` directory. The versions of each staged dependency are tracked in the [`scripts/tracked`](./tracked) file.
+
+## Setup
+
+The sync process requires the git-subtree command. See [git subtree](https://github.com/git/git/blob/master/contrib/subtree/INSTALL) for more detailed instructions.
+
+The local repo also needs to track the upstream remotes in [`scripts/tracked`](./tracked). To add these to your repo, run the `init_remotes` script from the root of your repo:
+```
+./scripts/init_remotes.sh
+```
```

Comment:
Same here, just a markdown linting check.

```suggestion
The local repo also needs to track the upstream remotes in [`scripts/tracked`](./tracked). To add these to your repo, run the `init_remotes` script from the root of your repo:

```bash
./scripts/init_remotes.sh
```
```

It took a quick look at that script - we may want to add a "force delete" type option that purges any existing api/operator-registry/operator-lifecycle-manager remotes in case you messed them up, but that's just QoL and nothing blocking right now.